proposal/MazenSli/proposal.rst

```diff
@@ -0,0 +1,76 @@
+===========================
+Proposal for Spline
+===========================
+
+Basic information
+=================
+
+It will be a branch of https://github.com/PhoenixDL/rising
+but can also be an extern standalone Repository, if required.
```

Comment:
It's OK to have your term project to be part of another open-source project, but you will need to make sure that by the end of the semester, when you present the project, your code is checked in to the upstream repository.

Having your code available publicly, as early as possible, will facilitate me to provide you comments in time.  If the code is in the upstream repository, please let me know how to access it.  If the code is not publicly available in the upstream repository, you need to provide your own so that I can access it publicly.